docs/user/admin.rst

```diff
@@ -147,6 +149,124 @@ Delete User’s Account
   :alt: confirm delete account
 
 
+Manage Classes
+--------------
+
+.. note::
+  To manage Kolibri classes and groups you must be logged-in as **Device Owner** or **Admin**.
+
+You can view, create and delete classes and learner groups, as well as search, filter and enroll Kolibri users in them, using the **Classes** tab in your **Manage** dashboard. Default view displays the list of all classes in your facility, with the number of enrolled users and assigned coaches for each class. 
```

Comment:
Will correct that in the next iteration of the docs.

Ref. the serial comma, it is mostly and AmE convention, and even if I write predominantly that variant, I do tend to drop the ball there often... :blush: